cluster/addons/dns/skydns-rc.yaml.in

```diff
@@ -28,12 +28,17 @@ spec:
             memory: 50Mi
         command:
         - /usr/local/bin/etcd
+        - -data-dir
```

Comment:
will split it out, makes sense because we might revert one. testing this was hard without data dir.